.gitignore

```diff
@@ -1,2 +1,4 @@
 tests/DoctrineTest/doctrine_tests/*
-*TestCase.php
\ No newline at end of file
+*TestCase.php
+/tests/tmp
+/tests/foo.sq3
```

Comment:
We may move foo.sq3 to /tests/tmp in (Doctrine_Connection_TestCase)

Reply:
Indeed, probably.

Or, even in memory.

Reply:
@alquerci can you create a issue to take care of this? thank you :)